src/isomorphic/modern/types/__tests__/ReactFlowPropTypes-test.js

```diff
@@ -1,18 +0,0 @@
-/**
- * Copyright 2014-present, Facebook, Inc.
- * All rights reserved.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree. An additional grant
- * of patent rights can be found in the PATENTS file in the same directory.
- *
- * @emails react-core
- */
-
-'use strict';
-
-describe('ReactFlowPropTypes', function() {
-
-  // TODO: Test Flow integration and ensure that prop types works.
```

Comment:
Jest now marks test files with no assertions as failing. I also really dislike having dead code in the codebase. If we are going to test proptypes with flow one day, we'll recreate this file.